src/catalog/database_catalog.cpp

```diff
@@ -759,6 +697,11 @@ table_oid_t DatabaseCatalog::GetTableOid(transaction::TransactionContext *const
 
 bool DatabaseCatalog::SetTablePointer(transaction::TransactionContext *const txn, const table_oid_t table,
                                       const storage::SqlTable *const table_ptr) {
+  auto *tm = txn->GetTransactionManager();
```

Comment:
Style nit: refer `txn_manager` over `tm`. I suspect this is actually all over the catalog.